homeassistant/components/bluetooth/__init__.py

```diff
@@ -102,11 +120,30 @@ def manufacturer_id(self) -> int | None:
 BluetoothCallback = Callable[[BluetoothServiceInfo, BluetoothChange], None]
 
 
+@hass_callback
+def async_discovered_devices(
```

Comment:
This should return BluetoothDiscoveryInfo from the history only if the device is still present